### PR TITLE
cssanduiupdates

### PR DESCRIPTION
**Pull request by haitzlm**
Update CSS and forms for better and cleaner display

### CODE REVIEW

**Comment by hortongn** [comment]
@haitzlm This PR has some login_mailer files that might not belong here?  They might be related to a login two-factor thing?

**Comment by haitzlm** [comment]
Do you want me to unstage them from here!??

**Comment by haitzlm** [comment]
The file [app/mailers/login_mailer.rb] has code for setting up a welcome email, though it is not configured yet. 
The file [app/views/login_mailer/login_mailer.text.html.erb] is the code for the email itself. 

I dont think it will hurt anything to merge these in, though they probably should have been on the mailerupdates branch

File: app/controllers/employees_controller.rb [hunk 1]
```diff
@@ -8,6 +8,11 @@ def index
     @employees = Employee.all
     # return unless params[:sort].present?
 
+    sort_column = params[:sort] || 'lastname'
```
Comment (hortongn) [inline app/controllers/employees_controller.rb:11]: Should this line be here?  It's already listed a few lines below.

**Review by hortongn — COMMENTED** [review]
@haitzlm See the comment I left in your change to the EmployeesController.  The CSS changes look good to me, but I'm not sure if you intend for the Staff Tools section to be pushed to the left like that (see screenshot).  Let me know.

Screenshot 2024-02-26 at 4 24 55 PM

**Review by hortongn — APPROVED** [review]
@haitzlm Looks good.  Squashing and merging.

**Pull request merged**